SAGESharp.Tests/Animations/TCBQuaternionDataTests.cs

```diff
@@ -25,7 +25,7 @@ public void Test_Getting_The_Value_Of_A_Component(ComponentTestCase testCase)
         [TestCaseSource(nameof(ComponentTestCases))]
         public void Test_Setting_The_Value_Of_A_Component(ComponentTestCase testCase)
```

Comment:
Same case in this test case, the assertion should have the value casted: `.Be((uint16)0x8000)`